src/components/Api.js

```diff
@@ -25,11 +38,11 @@ export default class Api {
     });
```

Comment:
` .then(this._checkResponse)` should be everywhere: after `deleteCard` and `dislikeCard` as well

You need to check if the response is `ok` everywhere